problems/problems.go

```diff
@@ -0,0 +1,147 @@
+package problems
```

Comment:
We still need to get this into the golibs i think.
And use the up-to-date logger here: https://github.com/seibert-media/golibs

Reply:
Moved `problems` to `problem` in `playnet-public/libs`